packages/shared/src/components/MainLayout.tsx

```diff
@@ -40,6 +35,17 @@ const Greeting = dynamic(
   () => import(/* webpackChunkName: "greeting" */ './Greeting'),
 );
 
+interface ShouldShowLogoProps {
+  mobileTitle?: string;
+  sidebarRendered?: boolean;
+}
+const shouldShowLogo = ({
+  mobileTitle,
+  sidebarRendered,
+}: ShouldShowLogoProps) => {
+  return !mobileTitle ? true : mobileTitle && sidebarRendered;
```

Comment:
I guess it should be like this?

```suggestion
  return !mobileTitle ? true : (mobileTitle && sidebarRendered);
```

Reply:
Prettier will remove those wrapping ()
So would say this is valid as it's inside a if else